src/main/java/org/osiam/addons/administration/controller/user/UserViewController.java

```diff
@@ -203,17 +207,101 @@ public String handleLimitAction(@RequestParam(value = REQUEST_PARAMETER_LIMIT) I
+				.addParameter(REQUEST_PARAMETER_LIMIT, session.getPagingInformation().getLimit())
+				.addParameter(REQUEST_PARAMETER_OFFSET, session.getPagingInformation().getOffset())
+				.addParameter(REQUEST_PARAMETER_ORDER_BY, session.getPagingInformation().getOrderBy())
+				.addParameter(REQUEST_PARAMETER_ASCENDING, session.getPagingInformation().getAscending())
+				.build();
+	}
+
+	@RequestMapping(params = REQUEST_PARAMETER_ACTION + "=activateSelected",method = RequestMethod.POST)
+	public String handlerUserActivationSelected (
+			@RequestParam(value = REQUEST_PARAMETER_USER_ID_ARRAY) final String[] id_array,
+			@RequestParam(value = REQUEST_PARAMETER_SEND_MAIL) final Boolean sendMail,
+			HttpServletRequest request) {
+
+		User user;
+		
+		for(String user_id : id_array){
+			userService.activateUser(user_id);
+			
+			if (sendMail) {
+				user = userService.getUser(user_id);
+				if(user.getLocale() == null) {
+					emailSender.sendActivateMail(user, request.getLocale());
+				} else {
+					emailSender.sendActivateMail(user);
+				}
+			}
+		}
+
+		return new RedirectBuilder()
+				.setPath(CONTROLLER_PATH)
+				.addParameter(REQUEST_PARAMETER_ACTIVATE_SELECTED_SUCCESS, true)
+				.addParameter(REQUEST_PARAMETER_QUERY, session.getPagingInformation().getQuery())
+				.addParameter(REQUEST_PARAMETER_LIMIT, session.getPagingInformation().getLimit())
+				.addParameter(REQUEST_PARAMETER_OFFSET, session.getPagingInformation().getOffset())
+				.addParameter(REQUEST_PARAMETER_ORDER_BY, session.getPagingInformation().getOrderBy())
+				.addParameter(REQUEST_PARAMETER_ASCENDING, session.getPagingInformation().getAscending())
+				.build();
+	}
+
+	@RequestMapping(params = REQUEST_PARAMETER_ACTION + "=deleteSelected", method = RequestMethod.POST)
+	public String handleUserDeletionSelected(
+			@RequestParam(value = REQUEST_PARAMETER_USER_ID_ARRAY) final String[] id_array,
+			HttpServletRequest request){
+
+		for(String user_id : id_array){
+			userService.deleteUser(user_id);
+		}
+
+		return new RedirectBuilder()
+			.setPath(CONTROLLER_PATH)
+			.addParameter(REQUEST_PARAMETER_DELETE_SELECTED_SUCCESS, true)
+			.addParameter(REQUEST_PARAMETER_QUERY, session.getPagingInformation().getQuery())
+			.addParameter(REQUEST_PARAMETER_LIMIT, session.getPagingInformation().getLimit())
+			.addParameter(REQUEST_PARAMETER_OFFSET, session.getPagingInformation().getOffset())
+			.addParameter(REQUEST_PARAMETER_ORDER_BY, session.getPagingInformation().getOrderBy())
+			.addParameter(REQUEST_PARAMETER_ASCENDING, session.getPagingInformation().getAscending())
+			.build();
+	}
+
+	@RequestMapping(params = REQUEST_PARAMETER_ACTION + "=deactivate", method = RequestMethod.POST)
 	public String handleUserDeactivation(
 			@RequestParam(value = REQUEST_PARAMETER_USER_ID) final String id,
 			@RequestParam(value = REQUEST_PARAMETER_SEND_MAIL) final Boolean sendMail,
 			HttpServletRequest request){
 
 		userService.deactivateUser(id);
-
+		
```

Comment:
you actually replaced the empty line with some white space. Please bring back the original, empty line.